NuGet.config

```diff
@@ -7,6 +7,8 @@
     <add key="dotnet-libraries-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries-transport/nuget/v3/index.json" />
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
+    <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
+    <add key="dotnet9-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9-transport/nuget/v3/index.json" />
```

Comment:
You will need to add these feeds to the package source mapping below.

Reply:
the official build is broken on dotnet/format and it is currently blocking the installer build

Reply:
https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=8562091&view=results

https://github.com/dotnet/installer/pull/17187